client/src/App.css

```diff
@@ -0,0 +1,32 @@
+.App {
+  text-align: center;
```

Comment:
If you are not using the CSS in this file, make sure to get rid it (for example the app logo boiler plate that came with the create-react-app